pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "backend-skeleton"
+version = "0.1.0"
+description = ""
+authors = ["DS <r@djangostars.com>"]
+readme = "README.md"
```

Comment:
Should we also remove `[build-system]` section? It seems that it is required only for packages, but I am not "poetry" expert 🤔

Reply:
I just checked, and it is working with the build section comment. Yes- we can remove it.
> I am thinking about make same changes to poetry in generated project
That is even more important than doing it there for the skeleton. Because these boring questions about authors always stack me)

Reply:
But I am not sure if it used for building "our" package or something from dependency 🤔